docs/sources/microsoft-365/outlook-cal/example-api-responses/sanitized_no-app-ids_no-groups/Events_v1.0.json

```diff
@@ -1036,12 +1000,9 @@
       "lastModifiedDateTime":"2021-02-25T01:31:48.0159305Z",
       "changeKey":"IiLKjG2I7E+Xv0+ys6MD0wADW1wcFg==",
       "categories":[ ],
-      "transactionId":null,
       "originalStartTimeZone":"UTC",
       "originalEndTimeZone":"UTC",
       "iCalUId":"040000008200E00074C5B7101A82E008000000004B36F3526728D3010000000000000000100000001E18A3027CF8964892BF097ACA3679E8",
-      "reminderMinutesBeforeStart":15,
```

Comment:
I'd include this on the unit test too, to ensure this is redacted when sanitized

Reply:
yeah, this is from the `sanitized` version used for tests. so should be covered by unit tests

Reply:
ok, but if the *original* file doesn't include that field we are not seeing the rule applied in the tests or in the files, but I was referring something like this:

https://github.com/Worklytics/psoxy/blob/6836d172f5de7dddcd48eecfdda21f4ed18b5638/java/core/src/test/java/co/worklytics/psoxy/rules/msft/CalendarTests.java#L35

so adding in the redacted assert the text  `reminderMinutesBeforeStart`, so at least we have something to check that prior the files we generate